edx_exams/apps/api/v1/views.py

```diff
@@ -421,6 +450,25 @@ class ExamAttemptView(ExamsAPIView):
 
     authentication_classes = (JwtAuthentication,)
 
+    def get(self, request):
+        """
+        HTTP GET handler to fetch all exam attempt data
+
+        Parameters:
+            None
+
+        Returns:
+            A Response object containing all `ExamAttempt` data.
+        """
+        user_id = request.user.id
+        attempt = get_latest_attempt_for_user(user_id)
+
+        if attempt is not None:
+            serialized_attempt = ExamAttemptSerializer(attempt)
+            return Response(data=serialized_attempt.data)
+
+        return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
I'm pretty sure we actually want to return an empty object here since if there's no active exam the request itself is still valid and we wouldn't want the UI to treat this as an error.

Reply:
By empty object, do you mean None, an empty array, or a serialized attempt with empty values?

Reply:
None or {}, I could argue for either really. Let's go with None?